src/hyperspectral/hyperspectral/target/matched_filter.py

```diff
@@ -0,0 +1,15 @@
+import math
+
+import numpy as np
+
+from hyperspectral.math.rpca import whitening_matrix, whiten
+
+def normalized_matched_filter(image, target, clutter_cov):
```

Comment:
It is straightforward to see what is happening, but a reference to why it is happening might be useful here.